tests/test_extractors.py

```diff
@@ -236,7 +236,9 @@ def test_nwb_extractor(self):
         # create new
         path2 = self.test_dir + '/firings_true.nwb'
         se.NwbSortingExtractor.write_sorting(self.SX, path2, session_description='second',
-                                                 identifier='19475')
+                                             identifier='19475')
+        SX_nwb = se.NwbSortingExtractor(path1)
```

Comment:
@luiztauffer this should be `path2`, but locally it then fails because `No acquisitions found in the .nwb file.`
You probably need to add a dummy acquisition if only spiketrains are saved

Reply:
you're right! we're working on it